index.html

```diff
@@ -0,0 +1,114 @@
+      <div class="text">
+        <h2>Hi, I'm Rami. Nice to meet you</h2>
+        <p>
+          From when I was a teenager I was interested in Web development, especially 
+          the front side of it since I prefer visuals and instantly seeing the changes
+          that I am doing. I'm currently doing an internship that is teaching me new things
+          and allowing me to sharpen my skills. After it, I would like to graduate from Computer
+          Science and start my career.
+        </p>
+      </div>
+    </section>
+
+    <div class="card-info">
+      <div class="card-info-1">
+          <img src="./images/skills.svg" alt="">
+          <h1>Front-end Developer</h1>
+          <p class = "p-personal">I like to code things from scratch, and enjoy bringing ideas to life in the browser.</p>
+          <h2>Languages I speak:</h2>
+          <p class = "p-skills">HTML, CSS, JavaScript</p>
+          <h2>Dev Tools:</h2>
+          <ul>
+            <li>Git</li>
+            <li>GitHub</li>
+            <li>Bootstrap</li>
+          </ul>
+      </div>
+    </div>
+        
+    <div class = "recent-work-container">
+      <h2 class="recent-work-container__h2">My Recent Work</h2>
+      <p class="recent-work-container__p">Here are a few design projects that I have worked on</p>
+
+      <div class="project-grid">
+          <div class="project-grid__1">
+            <img src="./images/starbucks.jpg" alt="starbucks-cup-image" class="project-img-1">
+            <div class="project-1-overlay project-1-overlay--primary ">
+              <div class="project-1-overlay__description">This project was a clone of Starbucks' main page </div>
+              <a href="https://github.com/Rami-ElSkakni/Starbucks-FrontEnd-Clone"><button class="project-1-overlay__button">Visit Website</button></a>
+           </div>
+          </div>
+          <div class="project-grid__2">
+            <img src="./images/cinema.jpg" alt="cinema" class="project-img-2">
+            <div class="project-2-overlay project-2-overlay--primary ">
+              <div class="project-2-overlay__description">View and sort movies that you like based on a database</div>
+              <a href="https://github.com/Rami-ElSkakni/DataBase-Frontend-Project"><button class="project-2-overlay__button">Visit Website</button></a>
+           </div>
+          </div>
+          <div class="project-grid__3">
+            <img src="./images/clothing.jpeg" alt="clothes" class="project-img-3">
+            <div class="project-3-overlay project-3-overlay--primary ">
+              <div class="project-3-overlay__description">A website that displays all kinds of clothing-wear</div>
+             <a href="https://github.com/Rami-ElSkakni/Clothing-Front-End-Website"><button class="project-3-overlay__button">Visit Website</button></a> 
+           </div>
+          </div>   
+    </div>
+
+    <footer class="footer">
+        <img class="footer__img" src="./images/mf-logo-white.svg" alt="">
```

Comment:
Dont make the class name like the tag.

Reply:
i do agree that this shouldnt be the case but in this specific footer one i think it makes sense